### PR TITLE
KEYCLOAK-18246 DemoFilterServletAdapterTest fails for app servers with TLS

### PR DESCRIPTION
**Pull request by mabartos**
JIRA: [KEYCLOAK-18246](https://issues.redhat.com/browse/KEYCLOAK-18246)

We can consider the _HTTPS_ listener is not present anymore and then we can ignore the exception.

### CODE REVIEW

**Comment by mabartos** [comment]
There are still a few failures after the first fix, so I pushed update fix and I'll share the results lately.

**Comment by mabartos** [comment]
Rebased and squashed commits. Tests passed in pipeline (see https://keycloak-jenkins.com/job/universal-test-pipeline-adapters/393/testReport/org.keycloak.testsuite.adapter.servlet/DemoFilterServletAdapterTest/)

@pdrozd @hmlnarik  Please, could you take a look at it? Thanks.

File: testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/ServerTestEnricherUtil.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
+package org.keycloak.testsuite.arquillian;
```
Comment (miquelsi) [inline testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/ServerTestEnricherUtil.java:1]: Missing the license text for the new file :-)

Comment (mabartos) [reply testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/ServerTestEnricherUtil.java:1]: @miquelsi Ahh, right!

Comment (mabartos) [reply testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/ServerTestEnricherUtil.java:18]: @miquelsi Done.

**Review by miquelsi — APPROVED** [review]
Thanks @mabartos, LGTM!

**Review by pdrozd — APPROVED** [review]

**Pull request merged**